//package com.my.exception;
//
//public class ExceptionTest6 {
//	public static void main(String[] args) {
//		String[] array = {"100", "1oo"};
//
//		for(int i=0;i<=array.length;i++) {
//			try {
//				int value = Integer.parseInt(array[i]);
//				System.out.println("array[" + i + "] : " + value);
//			}catch(Exception e) {
//				System.out.println("Exception 클래스 : " + e.getMessage());
//			}catch(ArrayIndexOutOfBoundsException e) {
//				System.out.println("ArrayIndexOutOfBoundsException 클래스 : " + e.getMessage());
//			}
//		}
//	}
//}
